### ISSUE TITLE
Base animations from TailwindCSS are gone

### ISSUE THREAD

**Issue by Seenko** [ikcb/animated-tailwindcss#171]
This package doesn't quite function how a normal TailwindCSS plugin does (you can't just register it inside the plugins array in your config), instead you have to wrap your own config with it's `withAnimations` function and that causes a problem. It gets rid of the default animations, such as `animate-spin`, `animate-pulse`, etc.

I believe this should be patched, maybe whenever this starts following Tailwind's conventions for plugins, but for now if you have the same issue, you can _yoink_ the data injected by their `withAnimations` function and extend the config yourself.

Here's a sample that worked for me, basically don't wrap their function (it doesn't require a config) and just use the response on your own config:

```javascript
const animatedTailwindCSS = require('animated-tailwindcss')();

module.exports = {
  theme: {
    extend: {
      animation: { ...animatedTailwindCSS.theme.animation },
      keyframes: { ...animatedTailwindCSS.theme.keyframes }
    },
  },
  plugins: [ ...animatedTailwindCSS.plugins ]
};
```

It's a bit dirty but it does the job for now.

**Comment by brc-dd** [comment]
This is actually mentioned explicitly in the [docs](https://ikcb.org/animated-tailwindcss/docs/installation):

> For compatibility reasons, the built-in animations (spin, ping, pulse, bounce) will be removed (or replaced by their Animate.css counterparts).

Earlier I was (internally) extending the config too, but that caused weird merging with the built-in ones like `bounce` as both TailwindCSS and Animate.css have it. IIRC, the keyframes were getting properties of both instead of Animate.css overriding TailwindCSS. Not sure if that issue still persists.

Regarding your hack, you can improve its readability by using destructuring assignment:

```js
const { theme, plugins: animationPlugins } = require('animated-tailwindcss')()

module.exports = {
  theme: { extend: { ...theme } },
  plugins: [ ...animationPlugins ]
}
```

We probably can have `tw-` prefixed default animations to avoid conflicts. I'll look into that. Thanks.

**Comment by Seenko** [comment]
> This is actually mentioned explicitly in the [docs](https://ikcb.org/animated-tailwindcss/docs/installation):

That is true! Although there isn't any proposed workaround for the time being, hence why I posted this issue, in case someone else comes up with a need to access the default animations.

> the keyframes were getting properties of both instead of Animate.css overriding TailwindCSS

That still persists, unfortunately, but for the built-in ones that I use I don't have any issues luckily!

> We probably can have `tw-` prefixed default animations to avoid conflicts.

One idea that will be less work and not having to bother transforming the defaults:

- Since this package isn't actually importing the `animate.css` package as a dependency, you could have a different name for the conflicting ones declared here, just document the animations in a playground style (like `animate.css` own website). It's already changing the default prefix `animate__` to `animate-` anyways.

**Comment by brc-dd** [comment]
> just document the animations in a playground style (like animate.css own website

This is already there: https://ikcb.org/animated-tailwindcss/docs/animations

**Comment by brc-dd** [comment]
Fixed in v3.3.0. Check the updated docs.
